style.css

```diff
@@ -1529,16 +1529,16 @@ ul li {
   top: calc(80% - 50px / 2);
 }
 
-.contact {
+.contact-background {
   position: absolute;
-  width: 375px;
+  width: 56vw;
   height: 779px;
-  left: 8px;
-  top: 4785px;
+  left: 0;
+  top: 4729px;
   background: #6070ff;
-  border-radius: 80px 0 0 0;
-  align-items: center;
+  justify-content: center;
   justify-items: center;
+  border-radius: 90px 0 0 0;
 }
```

Comment:
- [ ] Good work implementing an `ul` and `li` for the form section. Kindly fix your styling on it since when you are displaying the mobile version it's only visible the middle of the form.

- [ ] Kindly remove all the `position absolute` from your different elements on the contact form. To position the elements use display flex, you can learn more about this [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)

![image](https://user-images.githubusercontent.com/93630700/207445850-12cfc0b7-0b94-441d-ad46-93725c1bdb6c.png)